bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/gmos/GmosRecipe.java

```diff
@@ -301,14 +305,18 @@ private ImagingResult calculateImagingDo(final Gmos instrument) {
 
     // TODO: some of these warnings are similar for different instruments and could be calculated in a central place
     private List<ItcWarning> warningsForImaging(final Gmos instrument, final double peakPixelCount) {
-        final double wellLimit     = 0.95 * instrument.getWellDepth() * instrument.getSpatialBinning() * instrument.getSpectralBinning();
-        final double lowGainLimit  = 0.95 * instrument.getADSaturation() * instrument.getLowGain();
-        final double highGainLimit = 0.95 * instrument.getADSaturation() * instrument.getHighGain();
+        final double wellLimit = 0.95 * instrument.getWellDepth() * instrument.getSpatialBinning() * instrument.getSpectralBinning();
+        final double meanGain;
+        switch (gmosParameters.site()) {
+            case GN: meanGain = InstGmosNorth.getMeanGain(gmosParameters.ampGain(), gmosParameters.ampReadMode(), gmosParameters.ccdType()); break;
```

Comment:
It is, the problem is I don't have an spModel GMOS instrument object available here. I could create one, but instead I turned that method into a static method so I can use it from the ITC code without instantiating an spModel GMOS instrument.

(Ideally at some point we can combine the ITC instruments with the spModel ones, but I think that's going to be a long way to go..)